### ISSUE TITLE
minikube status says apiserver:stopped

### ISSUE THREAD

**Issue by cp99says** [kubernetes/minikube#14540]
### What Happened?

[logs.txt](https://github.com/kubernetes/minikube/files/9080697/logs.txt)

Please look into the issue and help me sort this out, I am new into kubernetes.
OS:- Cinnamon, minikube started using docker driver

### Attach the log file

When I check the status using minikube status, it says
minikube
type: Control Plane
host: Running
kubelet: Running
apiserver: Stopped
kubeconfig: Configured

**Comment by PalmaPedro** [comment]
Hello

same issue with me. I even allowed extra memory upon start by 'minikube start --memory=4g' but still the 'apiserver' is stopped. Have you fixed it?

**Comment by cp99says** [comment]
@PalmaPedro no, I was not able to solve the issue, I am still facing problems to start the cluster. I was fed up and started it in windows, there it ran in a single run. Hoping for some help from the Kube community :)

**Comment by PalmaPedro** [comment]
I'm also on Linux Mint, I'll report back if i find a solution

**Comment by cp99says** [comment]
@PalmaPedro I found a solution, and It worked for me. 
I installed minikube with docker, but this time without docker-desktop. Seems like docker-desktop might have some issue with its configuration and alignment with docker-ce and minikube.

First, I removed docker entirely from my system

-  sudo apt-get purge -y docker-engine docker docker.io docker-ce docker-ce-cli
-  sudo apt-get autoremove -y --purge docker-engine docker docker.io docker-ce  
-  sudo rm -rf /var/lib/docker /etc/docker
-  sudo rm /etc/apparmor.d/docker
-  sudo groupdel docker
-  sudo rm -rf /var/run/docker.sock
https://askubuntu.com/questions/935569/how-to-completely-uninstall-docker



Moreover, I followed this tutorial to install minikube and kubectl

-  https://linuxhint.com/install-minikube-ubuntu/
-  https://linuxhint.com/install-minikube-ubuntu/#b2

hope this helps

**Comment by PalmaPedro** [comment]
Thanks for the detailed answer, I appreciate it. In my attempts to setup minikube, I also realized that Docker desktop could be causing issues. I ended up trying to ditch docker as my driver to start minikube for now, and tried with virtualbox ( https://minikube.sigs.k8s.io/docs/drivers/virtualbox/ ). This seems to work for me for now, but i'll try as you suggest.

**Comment by klaases** [comment]
Hi @cp99says, yes, it sounds like removing and/or reinstalling Docker Desktop helped in this case, and I appreciate the follow up and commands/instructions.

With external dependencies, from time to time it appears that reinstallation is the appropriate "fix" for the issue.

**Comment by cp99says** [comment]
> Hi @cp99says, yes, it sounds like removing and/or reinstalling Docker Desktop helped in this case, and I appreciate the follow up and commands/instructions.
> 
> With external dependencies, from time to time it appears that reinstallation is the appropriate "fix" for the issue.

yes, but this should be mentioned somewhere either on the kubernetes website or docker-desktop webpage, that "due to dependency incompatibility, minikube wont even start " , so that newbies wont have to face this issue

**Comment by afbjorklund** [comment]
Docker Desktop on Linux is not yet supported by minikube

**Comment by cp99says** [comment]
sure, that would be a piece of helpful information to people who are getting started in the DevOps world and they don't need to waste time on setting the cluster up.

Thanks for the idea @klaases
